mne/io/nirx/nirx.py

```diff
@@ -19,15 +19,25 @@
 
 
 @fill_doc
-def read_raw_nirx(fname, preload=False, verbose=None):
+def read_raw_nirx(fname, nosatflags=False, preload=False, verbose=None):
     """Reader for a NIRX fNIRS recording.
 
-    This function has only been tested with NIRScout devices.
+    This function has only been tested with NIRScout and NIRSport1 devices.
 
     Parameters
     ----------
     fname : str
         Path to the NIRX data folder or header file.
+    nosatflags : bool
+        (Only relevant for NIRSport devices). Decide to use *.nosatflags_wlX
```

Comment:
Is this also relevant for NIRSport2 devices? If not, lets be specific and say NIRSport1.

Reply:
The documentation only talks about the v1, so I guess not. I'll be more specific